app/assets/stylesheets/title.scss

```diff
@@ -0,0 +1,22 @@
+$elegant: #f0efee;
+
+h1 {
+  font-family: "Avant Garde", Avantgarde, "Century Gothic", CenturyGothic, "AppleGothic", sans-serif;
+  font-size: 30px;
+  text-align: center;
+  text-transform: uppercase;
```

Comment:
Properties should be ordered font-family, font-size, text-align, text-rendering, text-transform